webapp/src/components/post_menu_actions/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -39,11 +42,15 @@ export default class AttachCommentToIssuePostMenuAction extends PureComponent {
     };
 
     connectClick = () => {
```

Comment:
This is actually why I had to change `sendEphemeralPost`, so the `index.js` parent component can correctly dispatch the action, since it doesn't have access to the actual redux store i.e. `{dispatch, getState}` that the hooks are using to dispatch the actions. I think the switch to use the `redux-thunk` way of doing this is low risk.

Reply:
This code is duplicated in the `create_issue` post_menu_action as well btw

Reply:
Yep, I noted that in the ticket.